src/apps/service/callbacks.clj

```diff
@@ -11,10 +11,13 @@
   (log/info (str "received a status update for DE job " uuid))
   (apps/update-job-status uuid))
 
-(defn update-agave-job-status
-  [job-id last-updated {:keys [status external-id end-time]}]
+(defn update-tapis-job-status
+  [job-id {{:keys [timestamp type data]} :event}]
   (service/assert-valid job-id "no job UUID provided")
-  (service/assert-valid status "no status provided")
-  (service/assert-valid external-id "no external job ID provided")
-  (log/info (str "received a status update for Agave job " external-id ": status = " status))
-  (apps/update-job-status job-id external-id status (first (remove string/blank? [end-time last-updated]))))
+  (service/assert-valid type "no status provided")
+  (let [{:keys [jobUuid]} (service/parse-json data)
```

Comment:
Not sure why once again, but this `data` field is JSON encoded as a string, so we have to parse it here in order to get the Tapis job ID.